rtf_vis_tool/README.md

```diff
@@ -0,0 +1,47 @@
+# GTSFM Pipeline Visualization Tool
+
+This project was created with React for viewing metrics related to the various processes within the GTSFM pipeline. Also used to view resulting point clouds using React Three Fiber.
+
+## Setup
+1. After cloning the repository, [install Node.js](https://nodejs.org/en/download/). Node.js serves as JavaScript runtime environment which the React application will run on.
```

Comment:
can Node be installed through conda, just by including it in the conda environment?

Reply:
sure, I'll look into this

Reply:
yup. I've added it to the `environment_linux.yml` file